### PR TITLE
Feature/add season modal

### PR DESCRIPTION
**Pull request by Jen-464**
### Description

This PR includes the Season Modal with new files: Seasons.tsx & SeasonForm.tsx. I've also added another file Seasons.module.css, but the styling is the same as Leagues.module.css. The createSeason POST function has been moved to service.ts.

### Issue Link

- Create Season Modal Ticket
[https://cascarita.atlassian.net/browse/C1-81](https://cascarita.atlassian.net/browse/C1-81?atlOrigin=eyJpIjoiNTk0OGM5NmJlMTk4NDAxN2E0YTQ4MDhkNzhiNGIyZGIiLCJwIjoiaiJ9)

### Checklist
I don't know how to do these checklists ;-;
- [ ] I have tested the changes locally by running `npm run test`
- [ ] I have added or updated relevant documentation
- [ ] I have autoformatted the code by running `npm run eslint`
- [ ] I have added test cases (if applicable)

### CODE REVIEW

**Comment by jipatino98** [comment]
@Jen-464 Before I finally review the PR, would you mind merging the latest changes of main to reflect the latest changes! This would include any additions and changes I made to some files, like `LeagueForm.tsx` and `Leagues.tsx`.

**Comment by Jen-464** [comment]
> @Jen-464 Before I finally review the PR, would you mind merging the latest changes of main to reflect the latest changes! This would include any additions and changes I made to some files, like `LeagueForm.tsx` and `Leagues.tsx`.

**Comment by Jen-464** [comment]
> @Jen-464 Before I finally review the PR, would you mind merging the latest changes of main to reflect the latest changes! This would include any additions and changes I made to some files, like `LeagueForm.tsx` and `Leagues.tsx`.

@jipatino98 I have updated my PR with all 11 updated commits as requested. Let me know if there are any remaining questions or updates needed to do.

File: server/config/config.json
Comment (jipatino98) [inline server/config/config.json:1]: I believe this file is not needed anymore so I will delete it from this PR

**Review by jipatino98 — APPROVED** [review]
Everything looks good for now! We can focus on integrating React Query in a separate PR.

**Pull request merged**